packages/office-ui-fabric-react/src/components/Nav/Nav.base.tsx

```diff
@@ -119,15 +118,8 @@ export class NavBase extends React.Component<INavProps, INavState> implements IN
         target={link.target}
         rel={rel}
         disabled={link.disabled}
-        aria-label={
-          link.ariaLabel && selectedStateAriaLabel
-            ? `${link.ariaLabel} ${selectedStateAriaLabel}`
-            : selectedStateAriaLabel
-            ? selectedStateAriaLabel
-            : link.ariaLabel
-            ? link.ariaLabel
-            : undefined
-        }
+        aria-current={isSelectedLink ? 'page' : undefined}
```

Comment:
We should introduce a new property on `link` that can be used to set this value; perhaps `ariaCurrentType` and set it to be an element in type `'page' | 'step' | 'location' | 'date' | 'time' | 'true' | 'false'`. There are several scenarios where a `Nav` can be used but doesn't actually control page navigation.

Reply:
That said, I think `page` is the perfect default if not otherwise provided.

Reply:
@jdhuntington is that something you'd like within this PR? I'd be happy to add `ariaCurrentType`, though I'd suggest limiting it to `'page' | 'step' | 'true'`.

Granted, I'm assuming the Nav component would not be used as a date grid, a list of times (where the times are not links), or a flowchart, (and that `undefined` is preferable to `false`). If those seem possible enough, or it doesn't seem like a good idea to restrict allowed values, the full list makes sense too :)

Reply:
@smhigley I think adding `ariaCurrent` prop to `INavLink` with the restrictions makes sense. If not provided defaults to 'page'.

Reply:
@dzearing done!